src/gui/editors/PianoRoll.cpp

```diff
@@ -4959,16 +4962,18 @@ PianoRollWindow::PianoRollWindow() :
 	// init edit-buttons at the top
 	auto editModeGroup = new ActionGroup(this);
 	QAction* drawAction = editModeGroup->addAction( embed::getIconPixmap( "edit_draw" ), tr( "Draw mode (Shift+D)" ) );
-	QAction* eraseAction = editModeGroup->addAction( embed::getIconPixmap( "edit_erase" ), tr("Erase mode (Shift+E)" ) );
+	QAction* eraseAction = editModeGroup->addAction( embed::getIconPixmap( "edit_erase" ), tr( "Erase mode (Shift+E)" ) );
```

Comment:
Our style guidelines usually suggest
```suggestion
	QAction* eraseAction = editModeGroup->addAction(embed::getIconPixmap("edit_erase"), tr("Erase mode (Shift+E)"));
```
However, since you did not intend to touch this code, I recommend just undoing this change and not touching this code. IMO, code should only change exactly those lines that are necessary.